config/example/node.env

```diff
@@ -56,11 +56,10 @@ NEOFS_CONTRACTS_PROXY=ad7c6b55b737b696e5c82c85445040964a03e97f
 # Morph chain section
 NEOFS_MORPH_DIAL_TIMEOUT=30s
 NEOFS_MORPH_DISABLE_CACHE=true
-NEOFS_MORPH_RPC_ENDPOINT="wss://rpc1.morph.fs.neo.org:40341/ws wss://rpc2.morph.fs.neo.org:40341/ws"
-
-# Main chain section (optional)
-NEOFS_MAINCHAIN_DIAL_TIMEOUT=30s
-NEOFS_MAINCHAIN_RPC_ENDPOINT="https://rpc1.n3.nspcc.ru:30341 https://rpc2.n3.nspcc.ru:30341"
+NEOFS_MORPH_RPC_ENDPOINT_0_ADDRESS="wss://rpc1.morph.fs.neo.org:40341/ws"
```

Comment:
could you, please, mention that config changes in the `CHANGELOG`?

Reply:
Fixed

Reply:
Fixed